alembic/versions/ae776dc8160b_add_koji_task_analysis_table.py

```diff
@@ -14,7 +14,7 @@
 
 # revision identifiers, used by Alembic.
 revision: str = "ae776dc8160b"
-down_revision: Union[str, None] = "2f60bb92411f"
+down_revision: Union[str, None] = "360aea89443f"
```

Comment:
```suggestion
down_revision: Union[str, None] = "d90c3ebde7ed"
```
I would have expected it to revise this version, but maybe you know something more that I don't.

Reply:
And if you have tested it locally and is working I am fine with the old value!

Reply:
The original patch was wrong. The field and comment should always have been `360aea89443f`